arm9/source/filesys/filetype.c

```diff
@@ -165,6 +166,9 @@ u64 IdentifyFileType(const char* path) {
         u64 type = 0;
         if ((fsize < SCRIPT_MAX_SIZE) && (strcasecmp(ext, SCRIPT_EXT) == 0))
             type |= TXT_SCRIPT; // should be a script (which is also generic text)
+	    // this should check if it's compiled lua bytecode (done with luac), which is NOT text
+	    else if ((fsize < LUASCRIPT_MAX_SIZE) && (strcasecmp(ext, LUASCRIPT_EXT) == 0))
```

Comment:
Very minor complaint, but looks like the indendation is not correct here.

Reply:
This was actually a mixture of tabs and spaces. It looked correct in my editor because of that so I never noticed.